tests/test_qa.py

```diff
@@ -0,0 +1,43 @@
+import argparse
```

Comment:
missing Intel header

Reply:
removed file since it's only running the procedure